src/components/sections/timeline.json

```diff
@@ -0,0 +1,25 @@
+{
+  "collectionName": "components_sections_timelines",
+  "info": {
+    "displayName": "Timeline",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
```

Comment:
Question:
Shouldn't we have theme for the section?

Reply:
The theme has been not mentioned in description. Let's discuss with Iryna, if we theme here.